src/form/Calendar/Calendar.module.css

```diff
@@ -4,6 +4,18 @@
     align-items: center;
     justify-content: space-between;
     margin-bottom: var(--spacing-sm);
+    padding-left: var(--spacing-md);
+    padding-right: 16px;
```

Comment:
we have to be strategic about when we affect styling like this - if it's for a new feature, we'll have to add it only to when the feature is turned on and as a token value such as this
```
.multi {
  padding: var(--multi-calendar-padding);
```

which gets added as an extra class when the feature is turned on. This is just an example of how you would go about adding styling.